### PR TITLE
cc-tool: new, 0.27

### PR DESCRIPTION
**Pull request by Fearyncess**
Topic Description
-----------------
cc-tool, introduce as a Texas Instrument's CC Debugger utilities.

Package(s) Affected
-------------------
`cc-tool`

Security Update?
----------------
No

Architectural Progress
----------------------
- [x] AMD64 `amd64`   
- [x] AArch64 `arm64`

Secondary Architectural Progress
--------------------------------
- [x] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`

----

After the pull request is merged, all package(s) affected must be rebuilt against the `stable` Git tree and environment (only `stable` repository should be enabled in `sources.list`). This section marks the progress above.

Please, make sure the list of architectures below matches the ones above.

Post-Merge Architectural Progress
---------------------------------
- [ ] AMD64 `amd64`   
- [ ] AArch64 `arm64`

Post-Merge Secondary Architectural Progress
-------------------------------------------
- [ ] Loongson 3 `loongson3`
- [ ] PowerPC 64-bit (Little Endian) `ppc64el`

### CODE REVIEW

File: extra-devel/cc-tool/autobuild/defines [hunk 1]
```diff
@@ -0,0 +1,6 @@
+PKGNAME="cc-tool"
+PKGSEC=devel
+PKGDES="A developing tool for Texas Instruments' CC Debugger"
```
Comment (MingcongBai) [inline extra-devel/cc-tool/autobuild/defines]: "Support library and tools for Texas Instruments CC Debugger"

**Review by MingcongBai — APPROVED** [review]

File: extra-devel/cc-tool/autobuild/defines (hunk 1 above)
Comment (MingcongBai) [inline extra-devel/cc-tool/autobuild/defines]: "Support library and tools for Texas Instruments CC Debugger"

**Review by MingcongBai — CHANGES_REQUESTED** [review]

File: extra-devel/cc-tool/autobuild/defines (hunk 1 above)
Comment (Fearyncess) [reply extra-devel/cc-tool/autobuild/defines]: `cc-tool` is only a tool for using CC Debugger. it doesnt provide any libraries.

Comment (MingcongBai) [reply extra-devel/cc-tool/autobuild/defines]: Then "Tools for Texas Instruments CC Debugger" or "Support tools for Texas Instruments CC Debugger"

**Comment by Fearyncess** [comment]
图片
cc-tool works well. it can flash the hex file to the chips

**Comment by MingcongBai** [comment]
@fearyncess Would you mind building for other architectures as well?

**Comment by Fearyncess** [comment]
i will try to build for other archs. but i think it depends on `libusb` compability.
________________________________
发件人: Mingcong Bai <notifications@github.com>
发送时间: 2020年11月17日 15:03
收件人: AOSC-Dev/aosc-os-abbs <aosc-os-abbs@noreply.github.com>
抄送: fearyncess <fsf@live.com>; Mention <mention@noreply.github.com>
主题: Re: [AOSC-Dev/aosc-os-abbs] cc-tool: new, 0.27 (#2516)


@fearyncess<https://github.com/fearyncess> Would you mind building for other architectures as well?

―

**Comment by MingcongBai** [comment]
@fearyncess 

> libusb compatibility.

Which is universal between architectures as a portable C library (and going as deep as the Kernel's USB stack).

**Review by MingcongBai — CHANGES_REQUESTED** [review]

**Comment by MingcongBai** [comment]
@fearyncess Ping.

**Review by MingcongBai — APPROVED** [review]

**Pull request merged**